docs/core/install/windows.md

```diff
@@ -97,11 +97,7 @@ While Visual Studio Code doesn't come with an automated .NET Core installer like
 
 The [download page](https://dotnet.microsoft.com/download/dotnet) for .NET provides Windows Installer executables.
 
-When you use the Windows installers to install .NET, you can customize the installation path by setting the `DOTNETHOME_X64` and `DOTNETHOME_X86` parameters:
-
-```console
-dotnet-sdk-7.0.100-win-x64.exe DOTNETHOME_X64="F:\dotnet\x64" DOTNETHOME_X86="F:\dotnet\x86"
-```
+With the release of arm64 support in 2021 November releases of .NET 3.1 and newer, custom msi install locations on Windows are no longer allowed and the `DOTNETHOME` environment variables are no longer supported. To install to a custom location, use the [dotnet-install scripts](../tools/dotnet-install-script.md).
```

Comment:
I think this could be clearer and simpler without "prior knowledge" nor calling out other versions.
```suggestion
> [IMPORTANT]
> Starting in November 2021, you can't change the installation path of .NET with the Windows Installer package. To install .NET to a different path, use the [dotnet-install scripts](../tools/dotnet-install-script.md).
```